doc/classes/Dictionary.xml

```diff
@@ -4,16 +4,63 @@
 		Dictionary type.
 	</brief_description>
 	<description>
-		Dictionary type. Associative container which contains values referenced by unique keys. Dictionaries are always passed by reference.
+		Dictionary type. Associative container which contains values referenced by unique keys. Just like in real life each dictionary composes multiple pairs of unique keys and values. You can define a dictionary by placing a comma separated list of key:value pairs in curly braces [code]{}[/code].
+		[codeblock]
+		var my_dir = {
+		    key1: value1,
+		    key2: value2,
+		    key3: value3
+		}
+		[/codeblock]
 		Erasing elements while iterating over them [b]is not supported[/b].
 		Creating a dictionary:
 		[codeblock]
-		var d = {4: 5, "A key": "A value", 28: [1, 2, 3]}
+		var my_dir = {} # Creates an empty dictionary.
+		var points_dir = {"White": 50, "Yellow": 75, "Orange": 100}
+		[/codeblock]
+		You can access values of a dictionary by refrencing approrpiate key in above example [code]points_dir["White"][/code] would return value of 50.
```

Comment:
refrencing -> referencing

Reply:
done

Reply:
approrpiate -> appropriate

Sorry, I should have caught that with the other typo.